admin/assets/javascripts/gds-pupil-form.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
where was this code defined/kept previously?

Reply:
@GuyHarwood I reailised `pupil-form.js` is not being used elsewhere so I removed `gds-pupil-form.js`. `pupil-form.js` contains the necessary changes now.